docs/source/index.rst

```diff
@@ -0,0 +1,23 @@
+Welcome to astrolabe's documentation!
```

Comment:
No, we don't have to publish elsewhere but I used RST because we use it for our specifications. At this point its more effort to switch it back so I'd be reluctant to do so unless there's some major problem with RST.

Reply:
Well my problem with RST is that we now need to publish this somewhere, right? Or is your intention that people just read this in the repo? My thought with converting to Markdown for the README.md is that it's implicitly published on the main page of the repo

Reply:
Yes, I will publish it using GH pages.